Extension/src/LanguageServer/client.ts

```diff
@@ -1962,7 +1962,7 @@ export class DefaultClient implements Client {
                     return;
                 }
                 const settings: CppSettings = new CppSettings(this.RootUri);
-                if (settings.configurationWarnings === "Enabled" && !this.isExternalHeader(docUri) && !vscode.debug.activeDebugSession) {
+                if (settings.configurationWarnings === "enabled" && !this.isExternalHeader(docUri) && !vscode.debug.activeDebugSession) {
```

Comment:
You'll need to force the value we have to lowercase before doing the comparison.